### ISSUE TITLE
Neural Network: Concept

### ISSUE THREAD

**Issue by chetandhembre** [chetandhembre/NN_Concepts#11]
# TODOs
- [x] [Activation Function](https://github.com/chetandhembre/NN_Concepts/blob/master/basics/Activation%20Function.md)
- [x] [Loss Function and Cost Function](https://github.com/chetandhembre/NN_Concepts/blob/master/basics/Loss%20Function%20and%20Cost%20Function.md)
- [x] [Regularization](https://github.com/chetandhembre/NN_Concepts/blob/master/basics/Regularization.md)
- [x] [Optimizer](https://github.com/chetandhembre/NN_Concepts/blob/master/basics/optimizer.md)
- [x] [Backpropagation](https://github.com/chetandhembre/NN_Concepts/blob/master/basics/Forward%20and%20Backward%20pass.md)
- [x] [Dropout](https://github.com/chetandhembre/NN_Concepts/blob/master/papers/dropout.md)

**Comment by chetandhembre** [comment]
Read this .. and include in your backprop write up
https://medium.com/@karpathy/yes-you-should-understand-backprop-e2f06eab496b
